tools/merge-databases/merge-databases.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# The purpose of this script is a one time data migration from
+# builder_originsrv, builder_sessionsrv and builder_jobsrv -> builder
+
+set -euo pipefail
+
+usage() {
+  echo "Usage: merge-databases.sh"
+  exit 1
+}
+
+if [ -z "${PGPASSWORD:-}" ]; then
+  echo "Please ensure that PGPASSWORD is exported in your environment and run this script again."
+  exit 1
+fi
+
+echo "This script will take all of the data that's currently in builder_originsrv, builder_sessionsrv and builder_jobsrv and merge them into builder."
```

Comment:
It might be worth saying `merge them into a brand new database, called "builder"`, or something like that.  As it reads right now, it could be confusing.